Tests/Event/MessageListEventTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of the AdminLTE bundle.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use KevinPapst\AdminLTEBundle\Event\MessageListEvent;
+use KevinPapst\AdminLTEBundle\Model\MessageModel;
+use KevinPapst\AdminLTEBundle\Model\UserModel;
+use PHPUnit\Framework\TestCase;
+
+class MessageListEventTest extends TestCase
+{
+    /**
+     * @test
```

Comment:
what does that annotation @test stand for?

Reply:
With that, I'm not forced to start the function with testThatBlaBla. I can just use a more fluent description.

Reply:
Thanks, didn't know that. Great to learn something new!